### DISCUSSION TITLE
YTSP for chrome

### DISCUSSION THREAD

**Discussion by rhythmichi** [ThioJoe/YT-Spammer-Purge#1055] [Ideas]
i really, REALLY want to use this but it does not support chrome! I dont think I need to explain more. 
except that i dont want it to be a extension.  I dont think a extension would be able to remove comments worldwide for everyone who views a video.

**Comment by Firecul** [comment]
I'm not sure what you mean by it not supporting Chrome, this is a stand alone application that only needs the web browser for completing the authorisation (for which Chrome is supported.)
This application can only remove comments if the signed in account is a moderator or owner of the channel the comment is found on, in which case yes it could remove it for everyone world wide. 
The main function of this app is allowing channel owners and moderators to effectively and quickly remove scam or spam comments across a large number of their own content. It's secondary function is allowing non-owners and moderators to be able to report these types of comments to Youtube so that *hopefully* they can update their own software to catch these for us.
If this was the only concern then an extension would work fine for that particular purpose, however, making this into an extension would be quite involved.
